app/routes.js

```diff
@@ -16,7 +16,7 @@ function banner(state, emit) {
 
 function body(main) {
   return function(state, emit) {
-    const b = html`<body class="flex flex-col items-center font-sans bg-blue-lightest md:h-screen md:bg-grey-lightest">
+    const b = html`<body class="flex flex-col items-center font-sans bg-blue-lightest md:h-screen md:bg-grey-lightest justify-between">
```

Comment:
When the promo banner is visible, on chrome or safari, `justify-between` causes a gap between the promo and the header. It's more pronounced the bigger the screen, or zoomed out.